### DISCUSSION TITLE
Medusa 2.0 setup

### DISCUSSION THREAD

**Discussion by bilalaboubi** [medusajs/medusa#7805] [Q&A]
Hello Medusa Community,

I am interested in installing Medusa 2.0 for a new project and would appreciate some guidance on the installation process. I've looked through the documentation and previous issues but couldn't find comprehensive instructions specific to version 2.0.

Could someone please provide a step-by-step guide or direct me to the relevant resources for installing Medusa 2.0? Any tips on prerequisites, potential pitfalls, or best practices would also be highly appreciated.

Thank you in advance for your help!

**Comment by vitalijalbu** [comment]
I think v2 isn’t mature yet to be used… now you may say: ok but they will release shortly and I’ll have the latest version already.

feel free to ask the official release date before.

ps. Also interested in v2💪🚀

**Reply by SalahAdDin** [reply]
It is not even released yet.

**Comment by Noiztbander** [comment]
As far as i know... they did not release V2 yet. So you cannot use it at this time. Im waiting too to migrate my projects to v2 but still waiting while they are working on the future release.

**Comment by blueholding** [comment]
you need nodejs server

**Reply by SalahAdDin** [reply]
Do we have any tutorial for it?
